core/deployment/src/main/java/io/quarkus/deployment/util/GlobUtil.java

```diff
@@ -0,0 +1,75 @@
+package io.quarkus.deployment.util;
+
+import java.util.regex.Pattern;
+
+/**
+ * @deprecated in favor of {@link io.quarkus.util.GlobUtil}
```

Comment:
Not related to this PR but I think it's a bad idea to use `io.quarkus.util` in the bootstrap project. Changes of having a split package later are very high.

Reply:
Do you want me to move it to `io.quarkus.bootstrap.util` or something?

Reply:
I don't know what the exact package should be. I had a look at the various bootstrap artifacts and they are very prone to end up with split packages as they all share the same namespace.

You would already have one but you have `util` in one and `utils` in the other :).

I'm not exactly sure if this can be fixed so late. Maybe we should talk about it on Zulip tomorrow?

Reply:
Yes, we may want to split the bootstrap-core in more modules in perspective. E.g. there is an overlap with dev tools.

Reply:
I don't think it's about splitting things in more modules. My concern is that we should have a properly separated root package for each of the existing modules.